pgml-dashboard/src/components/tables/serverless_models/mod.rs

```diff
@@ -0,0 +1,114 @@
+use crate::components::tables::small::row::Row;
+use pgml_components::component;
+use pgml_components::Component;
+use sailfish::TemplateOnce;
+
+#[derive(TemplateOnce, Default)]
+#[template(path = "tables/serverless_models/template.html")]
+pub struct ServerlessModels {
+    style_type: String,
+    embedding_models: [Component; 4],
+    instruct_models: [Component; 5],
+    summarization_models: [Component; 1],
+}
+
+impl ServerlessModels {
+    pub fn new() -> ServerlessModels {
```

Comment:
This feels like it belongs in our main app, not in the open source dashboard.

Reply:
it is used in our documentation.

Reply:
I understand. That requires us to fix our architecture to be able to inject data into the docs which may not be available in the open source repo.

Reply:
https://github.com/postgresml/postgresml/issues/1492